.github/workflows/zeebe-snyk.yml

```diff
@@ -152,7 +152,7 @@ jobs:
           # To avoid exiting on the first failure, we instead flip this to 1 as soon as one of the
           # command fails, and return this at the end; anything non 0 will cause the step to fail
           exitCode=0
-          JAVA_PROJECTS=(bom bpmn-model clients/java exporter-api gateway-protocol-impl protocol)
+          JAVA_PROJECTS=(bom clients/java spring-boot-starter-camunda-sdk zeebe/bpmn-model zeebe/exporter-api zeebe/gateway-protocol-impl zeebe/protocol)
```

Comment:
❌ This is wrong, spring sdk is under clients/ folder now

Reply:
Was my branch out of date? :see_no_evil: It's at the root for me

Reply:
I'll update it in a sec

Reply:
Thank you 👍